r2/r2/controllers/front.py

```diff
@@ -286,7 +286,8 @@ def GET_comments(
         infotext = None
         if request.GET.get('already_submitted'):
             submit_url = request.GET.get('submit_url') or article.url
-            resubmit_url = Link.resubmit_link(submit_url)
+            submit_title = request.GET.get('submit_title') or article.title
```

Comment:
I can see some use, although it doesn't seem too common.

If someone wants to resubmit an article, this way it would automatically supply the title as well.  But I see this as being very rare.  I feel like it could go either way.